### ISSUE TITLE
500 error trying to sign up

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/WebsiteOne#1788]
got this from a potential signup:

> I’m trying to create an account (not through Google) but I get all the time:

> 500 Internal Error

> Something went terribly wrong

haven't yet been able to find anything in the logs or papertrail or airbrake ...

**Comment by tansaku** [comment]
was able to sign up with the users email address on develop and staging but got 500 internal server error on production

```
2017-08-21T00:46:15.102900+00:00 app[web.1]: Started DELETE "/users/sign_out" for 133.202.198.78 at 2017-08-21 00:46:15 +0000
2017-08-21T00:46:15.116017+00:00 app[web.1]: Processing by Devise::SessionsController#destroy as HTML
2017-08-21T00:46:15.116061+00:00 app[web.1]:   Parameters: {"authenticity_token"=>"ZkYB4Oy1N7S4DMtBDjD+W8Vg1osXenQVDDiNTiPrTR8CDwqlzOnxP0zuCM2ZeNK2LJ+p7qt1BnhsKfdN0o5LIw=="}
2017-08-21T00:46:15.378948+00:00 heroku[router]: at=info method=POST path="/users/sign_out" host=www.agileventures.org request_id=4db719e7-4acd-41df-b405-a9a120cec791 fwd="133.202.198.78" dyno=web.1 connect=0ms service=282ms status=302 bytes=1063 protocol=https
2017-08-21T00:47:46.013481+00:00 heroku[router]: at=info method=GET path="/" host=www.agileventures.org request_id=aed94e1d-5cda-4245-b1c6-c9bd1752b4c9 fwd="133.202.198.78" dyno=web.1 connect=0ms service=1912ms status=200 bytes=7482 protocol=https
2017-08-21T00:47:54.044481+00:00 heroku[router]: at=info method=HEAD path="/" host=www.agileventures.org request_id=0536d784-4f90-4d0b-98a9-34eba39c81ad fwd="50.112.95.211" dyno=web.1 connect=0ms service=2ms status=301 bytes=143 protocol=http
2017-08-21T00:48:01.987592+00:00 app[web.1]: 
2017-08-21T00:48:01.987609+00:00 app[web.1]: Sent mail to info@agileventures.org (425.2ms)
2017-08-21T00:48:01.989534+00:00 app[web.1]:   Rendered static_pages/internal_error.html.erb within layouts/application (0.5ms)
2017-08-21T00:48:01.991029+00:00 app[web.1]:   Rendered layouts/_meta_tags.html.erb (0.8ms)
2017-08-21T00:48:01.993534+00:00 app[web.1]:   Rendered layouts/_head.html.erb (3.7ms)
2017-08-21T00:48:02.006147+00:00 app[web.1]:   Rendered layouts/_flash.html.erb (0.2ms)
2017-08-21T00:48:02.006302+00:00 app[web.1]:   Rendered layouts/_navbar.html.erb (12.5ms)
2017-08-21T00:48:02.009639+00:00 app[web.1]:   Rendered layouts/_sponsors.html.erb (2.9ms)
2017-08-21T00:48:02.015401+00:00 app[web.1]:   Rendered layouts/_footer.html.erb (5.5ms)
2017-08-21T00:48:02.015772+00:00 app[web.1]: Completed 500 Internal Server Error in 771ms (Views: 15.2ms | ActiveRecord: 27.3ms)
```

**Comment by tansaku** [comment]
but I was just able to sign up fine with a test email of my own devising ...

**Comment by tansaku** [comment]
the user does not exist already in the system ...

**Comment by tansaku** [comment]
It looks like they previously had an account that they created on 2017-06-01 13:13:57 and deleted on 2017-06-02 06:09:01

**Comment by tansaku** [comment]
ideally we'd display a more friendly message when we get a key violation like this ...

**Comment by praveendhawan** [comment]
This issue can be closed as this is resolved via #1631 .
